lib/opentracing.rb

```diff
@@ -0,0 +1,48 @@
+require "forwardable"
+require "concurrent"
+require "opentracing/version"
+require "opentracing/span_context"
+require "opentracing/span"
+require "opentracing/nil_tracer"
+
+module OpenTracing
+  FORMAT_TEXT_MAP = 1
+  FORMAT_BINARY = 2
+  FORMAT_RACK = 3
```

Comment:
please provide a pointer to the README about this one since it's not a standard OpenTracing thing.